examples/purchaseTesterTypescript/ios/Podfile.lock

```diff
@@ -408,7 +408,7 @@ DEPENDENCIES:
   - React-RCTVibration (from `../node_modules/react-native/Libraries/Vibration`)
   - React-runtimeexecutor (from `../node_modules/react-native/ReactCommon/runtimeexecutor`)
   - ReactCommon/turbomodule/core (from `../node_modules/react-native/ReactCommon`)
-  - RNPurchases (from `../../../`)
+  - RNPurchases (from `../../..`)
```

Comment:
how does this get set now that it's not in the podfile? is that what metro is doing 😅  sorry if this is a dumb question

Reply:
Nah! Not dumb question at all. This is auto linked with the introduction of the new `react-native.config.js` file. So whenever `pod install` is run, RN looks inside of `react-native.config.js` and will see this path and install it with cocoa pods